site/en/blog/longer-esw-lifetimes/index.md

```diff
@@ -35,11 +36,12 @@ Unfortunately, this behavior did not apply to extension events. Extension events
 
 ## What's changed
 
-As of Chrome 110, all events reset the lidle timer and the idle timeout will not occur if there are pending events. In other words, assuming there are no unexpected interruptions, extension service workers will now stay alive as long as they are actively processing events.
+As of Chrome 110, all events reset the lidle timer and the idle timeout will not occur if there are pending events. In other words, assuming there are no unexpected interruptions, extension service workers will now typically stay alive as long as they are actively processing events. Be aware that:
```

Comment:
I think it'd be great to elaborate a bit more on this:

- why are we're making this change?
- what was the previous behavior? (this helps devs gauge how this might impact the behavior of their extension)

Another thing: can we either remove `typically` or make it more precise? left on its own it is a bit confusing (at least I stumbled over it).

Reply:
>why are we're making this change?

Because premature termination was causing errors in people's code. At the time we figured that if somebody wasn't  having that problem they wouldn't be reading this article. Frankly, I'd like to err on the side of getting this out.

>what was the previous behavior?

It's described in the background.